amundsen_application/static/js/components/SearchPage/PopularTables/constants.ts

```diff
@@ -0,0 +1,4 @@
+// TODO: Hard-coded text strings should be translatable/customizable
```

Comment:
Consider changing the file path for this component under `homepage` or `common`. You can do this in a follow up after other related tasks might help determine where to put this.

Reply:
Will do this in a follow up PR as things are likely going to change